### DISCUSSION TITLE
Mount an init question

### DISCUSSION THREAD

**Discussion by cyberdwarf1664** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#11] [Ideas]
Hello, is it planned to add a mount tab on the character sheet and let the DM choose the dice used to initative (d20 instead of d10) ?

thanks

**Comment by aMediocreDad** [comment]
Hi Cyberdwarf1664,

Better supports for mounts is something we'd like to have. At the moment mounts are handled as Items (like the Forbidden Lands treats them as Gear). You can also make a character sheey to track their stats. That said, of higher priority is a proper NPC/Monster sheet that better supports the mechanical variety that exists. 

As to custom initiative. This is not something we are considering as it is homebrew and we'd like to have better support for the games mechanics before working on homebrew.

A temporary change is to edit the system.json file to include a "Initiative": "1d20" entry

**Reply by cyberdwarf1664** [reply]
I've managed the init with the init.js script.

Thanks for the answer and your work !
